src/main/java/com/playtika/automation/school/test/framework/action/NotesActions.java

```diff
@@ -0,0 +1,42 @@
+package com.playtika.automation.school.test.framework.action;
+
+import com.playtika.automation.school.test.framework.client.NotesFeignClient;
+import com.playtika.automation.school.test.framework.pojo.CreateNoteRequest;
+import com.playtika.automation.school.test.framework.pojo.UpdateNoteRequest;
+import com.playtika.automation.school.test.framework.pojo.responses.AuthResponse;
+import com.playtika.automation.school.test.framework.pojo.responses.NoteResponse;
+import lombok.AllArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+public class NotesActions {
+
+    private final NotesFeignClient notesFeignClient;
+
+    private String createAccessToken(AuthResponse responseData) {
```

Comment:
private methods should be placed after public and package-private and protected methods

Reply:
but after starting to use User object it can be gone